tools/binlog/deploy.md

```diff
@@ -0,0 +1,494 @@
+    2018/06/21 11:24:47 meta.go:117: [info] meta: &{CommitTS:400962745252184065}
+    ```
+
+    该命令会输出 `meta: &{CommitTS:400962745252184065}`，CommitTS 的值作为 Drainer 初次启动使用的 `initial-commit-ts` 参数的值。
+
+2. 修改 `tidb-ansible/inventory.ini` 文件
+
+    为 `drainer_servers` 主机组添加部署机器 IP，initial_commit_ts 请设置为获取的 initial_commit_ts，仅用于 Drainer 第一次启动。
+
+    - 以下游为 MySQL 为例，别名为 `drainer_mysql`。
+
+        ```ini
+        [drainer_servers]
+        drainer_mysql ansible_host=172.16.10.71 initial_commit_ts="402899541671542785"
+        ```
+
+    - 以下游为 file 为例，别名为 `drainer_pb`。
+
+        ```ini
+        [drainer_servers]
+        drainer_pb ansible_host=172.16.10.71 initial_commit_ts="402899541671542785"
+        ```
+
+3. 修改配置文件
+
+    - 以下游为 MySQL 为例
+
+        ```bash
+        $ cd /home/tidb/tidb-ansible/conf
+        $ cp drainer.toml drainer_mysql_drainer.toml
+        $ vi drainer_mysql_drainer.toml
+        ```
+
+        > **注意：** 
+        >
+        > 配置文件名命名规则为 `别名_drainer.toml`，否则部署时无法找到自定义配置文件。
+
+        db-type 设置为 "mysql"， 配置下游 MySQL 信息。
+
+        ```toml
+        # downstream storage, equal to --dest-db-type
+        # Valid values are "mysql", "file", "kafka", "flash".
+        db-type = "mysql"
+
+        # the downstream MySQL protocol database
+        [syncer.to]
+        host = "172.16.10.72"
+        user = "root"
+        password = "123456"
+        port = 3306
+        # Time and size limits for flash batch write
+        # time-limit = "30s"
+        # size-limit = "100000"
+        ```
+
+    - 以下游为增量备份文件为例
+
+        ```bash
+        $ cd /home/tidb/tidb-ansible/conf
+        $ cp drainer.toml drainer_pb_drainer.toml
+        $ vi drainer_pb_drainer.toml
+        ```
+
+        db-type 设置为 "file"。
+
+        ```toml
+        # downstream storage, equal to --dest-db-type
+        # Valid values are "mysql", "file", "kafka", "flash".
+        db-type = "file"
+
+        # Uncomment this if you want to use `file` or `sql` as `db-type`.
+        # `Compress` compresses the output file, like the `file` and `sql` file. Now it supports the `gzip` algorithm only. 
+        # The value can be `gzip`. Leave it empty to disable compression. 
+        [syncer.to]
+        compression = ""
+        # default data directory: "{{ deploy_dir }}/data.drainer"
+        dir = "data.drainer"
+        ```
+
+4. 部署 Drainer
+
+    ```bash
+    $ ansible-playbook deploy_drainer.yml
+    ```
+
+5. 启动 Drainer
+
+    ```bash
+    $ ansible-playbook start_drainer.yml
+    ```
+
+## 使用 Binary 部署 TiDB-Binlog
+
+### 下载官方 Binary
+
+```bash
+wget https://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz
+wget https://download.pingcap.org/tidb-{version}-linux-amd64.sha256
+
+# 检查文件完整性，返回 ok 则正确
+sha256sum -c tidb-{version}-linux-amd64.sha256
+```
+
+对于 v2.1.0 GA 及以上版本，Pump 和 Drainer 已经包含在 TiDB 的下载包中，其他版本需要单独下载 Pump 和 Drainer:
+
+```bash
+wget https://download.pingcap.org/tidb-binlog-{version}-linux-amd64.tar.gz
```

Comment:
before v2.1.0, binlog don't have version management. can only download the latest version

Reply:
Updated in https://github.com/pingcap/docs-cn/pull/1264/commits/e5747dc189987ae00864461b34130ce07f0d03c0